one_fm/one_fm/custom/salary_slip.json

```diff
@@ -0,0 +1,117 @@
+{
+ "custom_fields": [
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2021-11-24 23:42:18.029642",
+   "default": null,
+   "depends_on": "",
+   "description": "If Justification Needed on Deduction is True, Prepare Justification for PAM",
```

Comment:
I suggest you mention that it excludes `Social Security` component eighter in the checkbox naming or in the description to be clear for the users.